msg/en_US/csql.msg

```diff
@@ -164,6 +164,9 @@ $ csql help messages
    ;ROllback                    - roll back the current transaction.\n\
    ;AUtocommit [ON|OFF]         - enable/disable auto commit mode.\n\
    ;REStart                     - restart database.\n\
```

Comment:
;REStart                     - reconnect to the target database in a CSQL session.\n\

Reply:
Thank you. I will apply it.